src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const sourceObject = sourceArray.reduce((acc, curr) => {
+    const entry = curr.split(':').map((item) => item.trim());
+
+    acc[entry[0]] = entry[1];
+
+    return acc;
+  }, {});
```

Comment:
The variable name 'sourceObject' is too generic. Consider a name that indicates the purpose of the object, such as 'styleObject' or 'cssProperties'.